package.json

```diff
@@ -34,6 +34,7 @@
     "@web3-react/walletconnect-connector": "^6.1.1",
     "@web3-react/walletlink-connector": "^6.0.9",
     "ajv": "^6.12.3",
+    "bignumber.js": "^9.0.0",
```

Comment:
Adding `bignumber.js` for this , really ? 😅 
The exchange use JSBI from the pancake-sdk.

Reply:
I couldn't find how to use it for price it only contains integer functions as far as i see, can you give me an example?